lib/Notes/NoteCreatePage/NoteCreatePage.js

```diff
@@ -18,6 +18,7 @@ import { referredEntityDataShape } from '../components/NoteForm/noteShapes';
 @stripesConnect
 class NoteCreatePage extends Component {
   static propTypes = {
+    assignedContent: PropTypes.node,
```

Comment:
let's make it a function named `renderReferredRecordData`. `assignedContent` implies that we are passing something that will entirely replace the content of the "Assigned" accordion